src/components/TodoItem.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <li :style="[item.done ? {'background-color':'#698aab'} : {'background-color':'#a0d8ef'}]">
```

Comment:
It's better to use method wrapping up the style, or using [string literals](https://developer.mozilla.org/zh-TW/docs/Web/JavaScript/Reference/Template_literals)